### ISSUE TITLE
Giveth post in Octant's Epoch forum

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#1631]
From Matius in discord:

> Hello, it's wasn't made strictly required for Epoch Zero project but still you could create a discuss.octant.app post and put extra information there. You can look at some of the Epoch One projects submissions to get an idea (e.g. [rotki](https://discuss.octant.app/t/rotki-the-local-first-portfolio-management-tool-that-protects-your-privacy/39)). The format they used can be found in Step 3 [here](https://golem.foundation/2023/09/11/projects-submission.html#:~:text=Step%203%3A%20Complete%20Submission%20Form).
> 
> It'll be made a requirement for all projects Epoch Two on-wards, so sometime in the next few months it'll need to be created. Up to you when you'd like to do it 🙂

**Comment by laurenluz** [comment]
cc @SnakeGH @GriffGreen @Cotabe

**Comment by SnakeGH** [comment]
Thanks Lauren! @Cotabe since you are really taking charge of Octant, specifically the outreach right now, do you want to continue stewarding Octant and handling this "step 3" needs?

**Comment by Cotabe** [comment]
Yeah! I can take care of this. 

We can push more to update the UI in the groups with James & Matias. We have this doc: https://docs.google.com/document/d/1g0CNZjTCtEpHJZv2GPpev_bJP0RsrdSMkVH3WGjMW-U/edit

I don't think it makes much sense to create the forum post right now. I think it will take some time, it's not going to have much impact & most likely Giveth will have more updates and Octant might change the template for Epoch 3.

**Comment by laurenluz** [comment]
@brichis - we need to make a forum post & and update post in their forum for this...

https://docs.octant.app/submission-process.html#key-dates-and-deadlines

e.g.
https://discuss.octant.app/t/praise-the-standard-for-community-recognition/72/2

This needs to be done I think before Dec 20 - can you make a draft?
You can use the [octant description](https://docs.google.com/document/d/1g0CNZjTCtEpHJZv2GPpev_bJP0RsrdSMkVH3WGjMW-U/edit) for the initial forum post draft

and then for updates you can say we used some of the funds from epoch 1 for our Optimism QF round, and put in some of the stats of donations & projects & donors

and you can also then use our last [gitcoin grant description](https://docs.google.com/document/d/1TF5NsHh9agaPfn0UGc_ePULHOBYOhvAAjMhotNcBF5M/edit) to say the other things we "are using the money for" and "plan to use the money for"

Is that clear?

Basically also you should perhaps check the amount of funds we got from epoch 1.. consider that 50% went to the matching pool... and that basically we used only SOME of that funding for the Optimism round (because there was money that came from the other sponsors). you can kind of backwards math to get the exact amoubt of money from octant that went to funding the last QF round.

**Comment by brichis** [comment]
https://docs.google.com/document/d/1kJHnVPLGS6uuVJm5VMSo9uzk67VZz7gpxfpToAn_NUE/edit?usp=sharing 
wdyt @laurenluz ? I added the amount on top but I'm not sure how much USD that represents (for the exchange rate of the day)

**Comment by laurenluz** [comment]
Hey @brichis - I had a look, and I think it needs a lot of fixes. I left a bunch of comments.

Octant has a specific format for project posts, you can see examples in their forum, we should copy this format... not just do a copy/paste from our gitcoin grant.

also there needs to be more thought/explicit work going into explaining the impact that these funds really had... do some backwards math to show stats about our QF rounds and the direct impact of Octant's funds.

also I think refrain from mentioning work we're doing that we got other grant funding to do... unless that's ALL we're doing. I think we should focus on impressive work that we are doing that benefits projects overall, not just like optimism or superfluid.
